google/cloud/aiplatform/utils.py

```diff
@@ -194,7 +194,21 @@ def validate_project(project: str) -> bool:
     raise ValueError("Please provide a valid project ID")
 
 
-def validate_region(region: str):
+def validate_display_name(display_name: str):
+    """Verify display name is at most 128 chars and UTF-8 only.
+
+    Args:
+        display_name: display name to verify
+    Raises:
+        ValueError: display name is longer than 128 characters
+    """
+    if display_name is None:
```

Comment:
What would be the scenario when this is `None`? Also, if this is the case then the type hint should be `Optional[str]`.

Reply:
there are some methods that have optional display names. i've moved the None logic into those methods.